pyscriptjs/rollup.config.js

```diff
@@ -47,14 +61,7 @@ export default {
     }),
     // This will make sure that examples will always get the latest build folder
     !production && copy({
-      targets: [
-        { src: 'build/*', dest: 'examples/build' },
-      ]
-    }),
-    copy({
-      targets: [
-        { src: 'public/index.html', dest: 'build' },
-      ]
+      targets: copy_targets
     }),
     !production && serve(),
     !production && livereload("public"),
```

Comment:
These should be deleted as they are duplicate and the directory listing does not need to be livereload
```js
!production && serve(),
!production && livereload("public"),
```

Reply:
Agreed and removed.